packages/runtime/test/transport/messages.test.ts

```diff
@@ -272,6 +275,73 @@ describe("Message errors", () => {
         expect(result).toBeAnError("The recipient does not match the Request's peer.", "error.runtime.validation.invalidPropertyValue");
     });
 
+    test("should throw correct error message for trying to send a Message with a Request that expired", async () => {
+        const requestItem = {
+            "@type": "ConsentRequestItem",
+            consent: "I consent to this RequestItem",
+            mustBeAccepted: true
+        };
+
+        const expiresAt = CoreDate.utc().add({ milliseconds: 100 }).toString();
+        const createRequestResult = (
+            await client1.consumption.outgoingRequests.create({
+                content: {
+                    expiresAt,
+                    items: [requestItem]
+                },
+                peer: client2.address
+            })
+        ).value;
+
+        await sleep(150);
+
+        const result = await client1.transport.messages.sendMessage({
+            recipients: [client2.address],
+            content: createRequestResult.content
+        });
+
+        expect(result).toBeAnError(
+            "The Message cannot be sent as the contained Request is already expired. Please create a new Request and try again.",
+            "error.runtime.messages.cannotSendMessageWithExpiredRequest"
+        );
+    });
+
+    test("should mark a Request as expired when synced after the expiration date", async () => {
+        const requestItem = {
+            "@type": "ConsentRequestItem",
+            consent: "I consent to this RequestItem",
+            mustBeAccepted: true
+        };
```

Comment:
This `requestItem` can be omitted as well as it is already defined within the `beforeAll` block.